.rubocop.yml

```diff
@@ -44,7 +44,7 @@ Naming/AccessorMethodName:
 Style/Alias:
   Description: 'Use alias_method instead of alias.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#alias-method'
-  Enabled: false
+  Enabled: true
```

Comment:
I'd remove the lines 44-47 completely. Most cops are enabled by default (see [default.yml](https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml) in rubocop's code), this way DEV's .rubocop.yml file contains only the customized version of the configuration. What do you think @lightalloy ?

Reply:
But I think this way you can have control over which cops you want to enable even in the future don't you think?

Reply:
One perk of using the rubocop defaults is that as the community best practices change, those new best practice changes will trickle into the codebase with gem updates.

Reply:
@keshavbiswa that's what I mean, if you need to change the default, you just explcitly say so. I think all of these cops were added to the file exactly because they wanted do disable them

Reply:
Got it. Makes sense to remove these lines from rubocop.yml then.

Reply:
Agree with @rhymes on this one. 
@abraham changing best practices may be an issue, the possible solution is staying on a fixed version of `rubocop`. Currently `rubocop` is not automatically updated, and manual updates should be done regarding possible changing rules issue.